extensions/blocks/podcast-player/view.js

```diff
@@ -58,26 +69,228 @@ if ( window.jetpackPodcastPlayers !== undefined ) {
 // Replace the queue with an immediate initialization for async loaded players.
 window.jetpackPodcastPlayers = {
 	push: initializeBlock,
-	playerInstances,
 };
 
-// Add global handler for clicks.
-window.addEventListener( 'click', function( e ) {
-	// Prevent handling clicks if a modifier is in use.
-	if ( e.shiftKey || e.metaKey || e.altKey ) {
+function createSVGs() {
+	// for some reason document.createElement( 'svg' ) would not work. The HTML would work correctly, but the SVG would not render
```

Comment:
the `svg` element is namespaced: https://developer.mozilla.org/en-US/docs/Web/API/Document/createElementNS

```js
document.createElementNS( 'http://www.w3.org/2000/svg', 'svg' )
```

Reply:
Let's update this one in a follow-up PR.

Reply:
Fixed in https://github.com/Automattic/jetpack/pull/14989/commits/21692e1a5c24df313c1ff3ad93f3092b1d13063c